pom.xml

```diff
@@ -106,7 +106,7 @@
   <org.postgresql.postgresql.version>42.5.1</org.postgresql.postgresql.version>
 
   <com.github.spotbugs.version>4.7.3</com.github.spotbugs.version>
-  <com.github.spotbugs.plugin.version>${com.github.spotbugs.version}.0</com.github.spotbugs.plugin.version>
+  <com.github.spotbugs.plugin.version>4.7.3.2</com.github.spotbugs.plugin.version>
```

Comment:
@andrewazores What do you think of this change?

Reply:
Seems fine to me.